### ISSUE TITLE
feature don't respect override version of bundle

### ISSUE THREAD

**Issue by hieplq** [reficio/p2-maven-plugin#128]
I modify version of artifact by override, everything fine but on feature version is not update, it use old value

`							<artifacts>
								<artifact>
									<id>org.javassist:javassist:${zk-javassist-src-version}</id>
									<source>false</source>
									<transitive>false</transitive>
									<override>true</override>
									<instructions>
										<Bundle-SymbolicName>zjavassist</Bundle-SymbolicName>
										<Bundle-Name>zjavassist</Bundle-Name>
										
										
										<Bundle-Version>${zk-version}</Bundle-Version>
									</instructions>
								</artifact>
							</artifacts>`

**Comment by ivy-rew** [comment]
Yes I'm suffering of the same issue. If <instructions><Bundle-Version/> is set, the plugin reference in the feature should contain the same version string....
